examples/plot/plot-layer/axes-layer.js

```diff
@@ -254,23 +256,42 @@ export default class AxesLayer extends Layer {
     attribute.value = new Float32Array(flatten(normals));
   }
 
+  calculateInstanceIsTitle(attribute) {
+    const {ticks} = this.state;
+
+    const isTitle = ticks.map(axisTicks => {
+      const ticksCount = axisTicks.length - 1;
+      return axisTicks.map((t, i) => i < ticksCount ? 0 : 1);
+    });
+
+    attribute.value = new Float32Array(flatten(isTitle));
+  }
+
   // updates the instancePositions and instanceNormals attributes
   calculateTicks(bounds, ticksCount) {
     const xTicks = getTicks(bounds[0], ticksCount);
     const yTicks = getTicks(bounds[1], ticksCount);
     const zTicks = getTicks(bounds[2], ticksCount);
 
-    return [xTicks, yTicks, zTicks];
+    return [
+      [...xTicks, (bounds[0][0] + bounds[0][1]) / 2],
```

Comment:
Not yet. The solution to axes scaling was implemented by @Pessimistress probably on some other branch.